### PR TITLE
#1213 support for syncing reflections

### PR DESCRIPTION
**Pull request by alexfjw**
Part of #1213

### **Outline of Solution**

- Adds support for reflections to SyncLab.

The preview image for the reflection item in SyncLab has been emphasized by using a fixed transparency, blur and shadow size.

### CODE REVIEW

**Comment by junkiattan** [comment]
Tested on PPT2013/2016 and codes works as intended. Test provided also passes

Code LGTM

@leeyh20 Ready to check on 2010

**Comment by leeyh20** [comment]
Works on PPT 2010.

File: PowerPointLabs/PowerPointLabs/SyncLab/ObjectFormats/ReflectionEffectFormat.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            try
+            {
+                ReflectionFormat srcFormat = formatShape.Reflection;
+                ReflectionFormat destFormat = newShape.Reflection;
+
+                if (srcFormat.Type != MsoReflectionType.msoReflectionTypeMixed)
+                {
+                    // setting ReflectionFormat.Type automatically sets Reflection settings
+                    // there is no need to set them manually
+                    destFormat.Type = srcFormat.Type;
```
Comment (IanTeo) [inline PowerPointLabs/PowerPointLabs/SyncLab/ObjectFormats/ReflectionEffectFormat.cs]: User can actually customize the reflection after setting the default one. Is this handled here? As seen in the else block below, there are actually 4 things that the user can configure. What do you think about allowing them to sync them each individually?

**Comment by initialshl** [comment]
Good to have a peer code review before i review this

File: PowerPointLabs/PowerPointLabs/SyncLab/ObjectFormats/ReflectionTransparencyFormat.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            if (!Sync(formatShape, newShape))
+            {
+                Logger.Log(newShape.Type + " unable to sync ReflectionTransparency format");
+            }
+        }
+
+        public override Bitmap DisplayImage(Shape formatShape)
+        {
+            return SyncFormatUtil.GetTextDisplay(
+                Math.Round(formatShape.Reflection.Transparency * 100).ToString() + "%",
```
Comment (leeyh20) [inline PowerPointLabs/PowerPointLabs/SyncLab/ObjectFormats/ReflectionTransparencyFormat.cs]: Do you think it is needed to explain why need to multiply by 100?

Comment (alexfjw) [reply PowerPointLabs/PowerPointLabs/SyncLab/ObjectFormats/ReflectionTransparencyFormat.cs]: Hmm.. the docs do say that it's a float between 0 & 1.
And we're displaying as a percentage so.. maybe not?

Comment (leeyh20) [reply PowerPointLabs/PowerPointLabs/SyncLab/ObjectFormats/ReflectionTransparencyFormat.cs]: Oh if the docs mention then it's fine then!

Comment (alexfjw) [reply PowerPointLabs/PowerPointLabs/SyncLab/ObjectFormats/ReflectionTransparencyFormat.cs]: Cancel that, the docs are a bit ambiguous. I'll update that in a bit

**Pull request merged**